<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QWidget" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Direction</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Envoyer une capture d'écran au LEDCube.&lt;/p&gt;&lt;p&gt;Envoi possible à deux serveurs simultanément.&lt;/p&gt;&lt;p&gt;La capture d'écran est compressée en 8x8 et affichée sur le LEDCube.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lAddresse1">
     <property name="text">
      <string>ws://localhost:80/ws</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QCheckBox" name="checkMiroirV">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QSpinBox" name="sLayerIndex">
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>7</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Rotation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="lPassword1">
     <property name="text">
      <string>secretLedcube</string>
     </property>
    </widget>
   </item>
   <item row="12" column="2">
    <widget class="QCheckBox" name="checkLumAuCarre">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="lPassword2">
     <property name="text">
      <string>secretLedcube</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QSpinBox" name="sDelay">
     <property name="minimum">
      <number>10</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Délais d'envoi (ms)</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QComboBox" name="cbLayerDirection">
     <property name="currentText">
      <string>Y</string>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <item>
      <property name="text">
       <string>X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Z</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>Luminosité au carré</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="bConnect1">
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>(plus de contraste)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lAddresse2">
     <property name="text">
      <string>ws://ledcube.local:80/ws</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QComboBox" name="cbLayerRotation">
     <property name="currentText">
      <string>0</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>90</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>180</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>270</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Serveur 1</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Miroir vertical</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Indice de couche</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QCheckBox" name="checkMiroirH">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="bConnect2">
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Serveur 2</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Miroir horizontal</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>Facteur de saturation</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="QDoubleSpinBox" name="sSaturationFactor">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.500000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
